css/src/components/tabs.scss

```diff
@@ -0,0 +1,110 @@
+$tabs-padding: 0.5em 1.25em !default;
+$tabs-padding-large: 1em 1.5em !default;
```

Comment:
nit: large -> `lg` for variable matching class

Reply:
updated